make/migrations/postgresql/0005_ng_replication.up.sql

```diff
@@ -0,0 +1,18 @@
+/*
+  TODO: Alter existing table 'target', instead of creating new one.
+ */
+CREATE TABLE registry (
+ id SERIAL PRIMARY KEY NOT NULL,
+ name varchar(256),
+ url varchar(256),
+ credential_type varchar(16),
+ access_key varchar(128),
+ access_secret varchar(1024),
+ type varchar(32),
+ insecure boolean,
+ description varchar(1024),
+ health varchar(16),
+ creation_time timestamp default CURRENT_TIMESTAMP,
+ update_time timestamp default CURRENT_TIMESTAMP,
+ CONSTRAINT unique_url UNIQUE (url)
```

Comment:
The unique is not necessary, uses can create two registries with the same url but different credentials. The name should be unique

Reply:
OK, I will change it.